docs/fast_data/configuration/single_view_creator.md

```diff
@@ -142,15 +142,27 @@ service.addHook('onClose', async() => {
 | SV_UPDATE_VERSION                   | -        | (v6.2.1 or higher) Define which version of the `sv-update` event should be emitted by the service. Accepted values are `v1.0.0` and `v2.0.0`. By default, for retro-compatibility, version `v1.0.0` is employed                                                                                                                                                                                                                                                                                                             | v1.0.0              |
-
-If you do not want to use Kafka in the Single View Creator, you can just not set the environment variable *KAFKA_CLIENT_ID* or *KAFKA_BROKERS*. If one of them is missing, Kafka will not be configured by the service (requires *single-view-creator-lib* `v9.1.0` or higher)
+| KAFKA_SV_RETRY_TOPIC                | -        | Topic name for the [Single View Retry](#single-view-retry) mechanism. This enables the system which uses a Kafka consumer and a Kafka producer.                                                                                                                                                                                                                                                                                                                                                                             | -                   |
+| KAFKA_SV_RETRY_MAX_ATTEMPTS         | -        | Number of attempts allowed for a failed Projection Change. Defining `0` will **not** retry the failed aggregations but **will send the retry message to the KAFKA_SV_RETRY_TOPIC**                                                                                                                                                                                                                                                                                                                                          | 5                   |
+| KAFKA_SV_RETRY_DELAY                | -        | Time in milliseconds in which the service buffers the retry messages before sending them. This is meant to reduce the impact on performace of sending the retry messages.                                                                                                                                                                                                                                                                                                                                                   | 5000                |
+
+:::caution
+If you want to enable any mechanism that uses Kafka in the Single View Creator, like the [Single View Patch](/fast_data/configuration/single_view_creator.md#single-view-patch), remember to declare the following environment variables: 
```

Comment:
Could we mention first the SV Retry (I mean, before "like the Single View Patch")? Just to put more emphasizes on the former mechanism.

What do you think?

Reply:
Sound good, done!